test/reshape.jl

```diff
@@ -507,4 +507,58 @@ end
     @test eltype(typeof(sdf2.value)) === Float64
 end
 
+@testset "permutedims" begin
+    df1 = DataFrame(a=["x", "y"], b=rand(2), c=[1,2], d=rand(Bool,2))
```

Comment:
it would be easier to read tests if you followed the data frame definition immediately by the tests using it.